templates/shortcodes/crypto_table.md

```diff
@@ -2,43 +2,15 @@
 
 {{ trans(key='name', lang=lang) }} | {{ trans(key='token', lang=lang) }} | {{ trans(key='address', lang=lang) }}
 :-------------|:-----|:----------------------------------------------------------
-Algorand      | ALGO | 3UZXZEPGFAM7E74IO32Y7O6KOY3E7NNNJVBV4GFS5UWQQSY7AIM5PK7C2E
-Avalanche C/ETH | AVAX | 0x1D59bBe5d4332e34116DccDE5c1a8c736E1C2810
-Binance Coin/BNB | BNB  | bnb1c8vmnqqhqc6lcajpzvuy4ss5dw3vjc7tc5q8zd
-Binance Coin/BSC | BSC  | 0x1D59bBe5d4332e34116DccDE5c1a8c736E1C2810
-Binance USD/ETH |BUSD | 0x1D59bBe5d4332e34116DccDE5c1a8c736E1C2810
-Bitcoin       | BTC  | bc1qjkq3tpy2gutsfdlcvys8slkempywk230u8rc8u
-Bitcoin Cash  | BCH  | qqcjkzf7nlgvhng5vhq7n7hjrcdqccyqlq9h7gq4xw
-Cardano       | ADA  | addr1qxh59080ujswxuzapzrdvuzpglfktg09gq9q7dxpdl7jfka0g27wle9qudc96zyx6ecyz37nvks72sq2pu6vzmlayndsj02qhw
-Chainlink/ETH | LINK | 0x1D59bBe5d4332e34116DccDE5c1a8c736E1C2810
-Chainlink/SOL | LINK | Eyv43vXxmSshDPReJnfMtrDspugsVR3S6PzJV38rMAZE
-Cosmos        | ATOM | cosmos1jurqcft6vf8gah8lpyz53ue9qf5gz2rfy8wfla
-Dash          | DASH | XgWTXGVCNEVVEkL9DQutscjcs6zvDUgccf
-Dai/ETH       | DAI  | 0x1D59bBe5d4332e34116DccDE5c1a8c736E1C2810
-Dogecoin      | DOGE | DRdtRetmiSFHLkorNHyf4nL2MT375Xkmrm
-eCash         | XEC  | ecash:qpy6c5050vyste69tph2xmmqjchwldvggg7pe99937
-Ethereum      | ETH  | 0x1D59bBe5d4332e34116DccDE5c1a8c736E1C2810
```

Comment:
These wallets don't hurt. Maybe hide them into the summary/details tags?
<details>
<summary>More coins here</summary>
Detailed list of other coins
</details>